src/firebaseConfig.ts

```diff
@@ -5,74 +5,146 @@
 // 'npm run generate:json-schema' to regenerate the schema files.
 //
 
-export type DatabaseConfig =
+// Sourced from - https://firebase.google.com/docs/hosting/full-config#rewrite-cloud-run-container
+type CloudRunRegions =
```

Comment:
Keeping up with ever-expanding Cloud region launches is consistently hard for us. I know this is more typesafe but I would prefer to actually just use `"string"` for these fields and leave the validation to a runtime concern which can be validated via API calls. That way we don't have to constantly manually update this list as new regions launch.

Reply:
I was trying to pull this from elsewhere in the code, as with runtimes list.

I know maintenance will be difficult if not shared within this codebase, I would urge this should be a goal to strive for in future.